crates/sui-core/src/authority_aggregator.rs

```diff
@@ -871,6 +874,7 @@ where
                                             response.err().map(|err| (name, err))
                                         })
                                         .collect(),
+                                    action: "get_object_by_id".to_string(),
```

Comment:
Is there a way the action field can be a `&str`?  You see this is very inefficient, in that we have to create a new string copy for what should be a CONST or static string.

I mean I know this is just error, but trying to be mindful of where we can save allocations.  :)

Reply:
replace with `&'static str`